### ISSUE TITLE
Google Cloud Installation JSON File Question

### ISSUE THREAD

**Issue by ricemark9** [jlund/streisand#581]
I'm having the same issue as seen in #472 where I need to put the JSON file in the right location.  I understand that why the installation fails but I don't know how to fix it.  I'm inexperienced in all of this and I'm trying to figure out how to put the file in the correct location.  When I leave it blank it indicates there's no file  at /home/userx/streisand.json and was just wondering how to get it there.  I appreciate any help.

**Comment by CorbanR** [comment]
What OS are you using? OSX, Ubuntu, etc.


If your using OSX The default download location is `~/Download`. In this scenario, it should be as easy as opening up a terminal and typing `mv ~/Downloads/<project_name>.json ~/streisand.json`

**Comment by ricemark9** [comment]
I am using OS X and was able to get the JSON file in the right location.  Thank you for the help.  It now fails two steps later and it appears something is wrong with the google.yml file.  I thought I had set everything up correctly with applying a service address to the VM instance in GCE but now I'm not so sure.  I've included a screenshot if that helps.


<img width="1222" alt="screen shot 2017-04-08 at 23 45 18" src="https://cloud.githubusercontent.com/assets/27033729/24834505/d03d6536-1cb5-11e7-8f37-8b3a8e884814.png">
